python/ray/serve/schema.py

```diff
@@ -978,6 +978,63 @@ class ReplicaDetails(ServeActorDetails, frozen=True):
     )
 
 
+@PublicAPI(stability="alpha")
+class MetricsHealth(str, Enum):
```

Comment:
Better to be more precise about what this object represents 

```suggestion
class AutoscalingMetricsHealth(str, Enum):
```

Reply:
Thanks, addressed!